element-template-generator/http-dsl/src/main/java/io/camunda/connector/generator/dsl/http/OperationParseResult.java

```diff
@@ -14,12 +14,11 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package io.camunda.connector.generator.openapi;
+package io.camunda.connector.generator.dsl.http;
```

Comment:
One last thing - the PR title should follow Conventional Commit formatting

(I see how my original comment might have been confusing - my intended point was to add a noun after "unified" 😄 )

Reply:
Oh yeah sorry, I didnt know the PR name should also follow this pattern doing it right now 😅